02-basics-2/50-selected-meetup/SelectedMeetupApp.js

```diff
@@ -1,75 +1,61 @@
-import { defineComponent } from 'vue'
-// import { getMeetup } from './meetupsService.ts'
+import { defineComponent, ref, onMounted, watch } from 'vue'
+import { getMeetup } from './meetupsService.ts'
 
 export default defineComponent({
   name: 'SelectedMeetupApp',
 
-  setup() {},
+  setup() {
+    const checkedId = ref(1)
+    let meetupTitle = ref('')
```

Comment:
Defining a `ref` variable with `let` is almost always a mistake:
- Either this variable is never changed => it should be `const`
- Or a variable value is changed instead of a ref's value via `.value`